pkg/bsp-imx/patches/uboot-patches-lf_v2022.04/0013-dts-Add-mcu_rdc-node-to-imx8mp-rsb3720-a1-u-boot.dts.patch

```diff
@@ -0,0 +1,85 @@
+From b90a60af24e92118079d8d28273df14efd52e909 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Ren=C3=AA=20de=20Souza=20Pinto?= <rene@renesp.com.br>
+Date: Wed, 11 Jan 2023 14:33:10 +0100
+Subject: [PATCH 13/17] dts: Add mcu_rdc node to imx8mp-rsb3720-a1-u-boot.dtsi
```

Comment:
Are the patches which say "Subject: [PATCH x/y]" existing patches pulled from somehere?
I see all but one are signed off by you, which is fine if you had to modify the patches, but it would be good to also indicate the origin of the patches if you didn't do them from scratch.
And for unmodified patches it makes sense to keep the original signed-off-by.

Reply:
This notation it's because I do my changes in the kernel, commit them to my custom branch and at the end I use `git format-patch` to generate the patches for EVE. At some extent it indicates how many patches I've added, but I could agree it might be confused, do you think is worth to remove them?

Reply:
> I see you added ledmanager support for the user LED which is great.
> 
> The other thing to do separately is to generate a hardware model json and submit a PR to eden to add that to the models directory.
> 
Ok, will do it.

> Suggestions for who is the best person to review the details of the patches?
I think @rouming, @mikem-zed and @OhmSpectator can be the reviewers for this PR... and if anybody else wants to join will be very welcome...